03 - Playing with PyPI.ipynb

```diff
@@ -388,7 +388,7 @@
    "metadata": {},
```

Comment:
<p>This one took couple of seconds for me to load, not sure if that's ok but just as an FYI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>